packages/load-script/src/index.js

```diff
@@ -48,15 +48,15 @@ export function loadScript( url, callback ) {
 	} );
 }
 
-export function loadjQueryDependentScript( url, callback ) {
+export function loadjQueryDependentScript( url, callback, args ) {
 	debug( `Loading a jQuery dependent script from "${ url }"` );
 
 	if ( window.jQuery ) {
 		debug( `jQuery found on window, skipping jQuery script loading for "${ url }"` );
-		return loadScript( url, callback );
+		return loadScript( url, callback, args );
 	}
 
-	const loadPromise = loadScript( JQUERY_URL ).then( () => loadScript( url ) );
+	const loadPromise = loadScript( JQUERY_URL ).then( () => loadScript( url, null, args ) );
```

Comment:
Is there a good reason to specifically pass `null` here instead of `callback`? 

```suggestion
	const loadPromise = loadScript( JQUERY_URL ).then( () => loadScript( url, callback, args ) );
```

In the case when `callback` is not specified it will be `undefined` and not `null`.